lib/resources/example_json/r4_examples_encounter.rb

```diff
@@ -433,12 +433,13 @@ module Resources
           "type": {
             "coding": [
               {
-                "system": "http://hl7.org/fhir/v2/0203",
-                "code": "FIN NBR"
+                "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/319",
```

Comment:
I'd prefer not to have this (writing a FIN) listed in the actual example. I think it's fine to have it in the body fields to explain what you could do with the field, but not list it out in the example.

My concern is that FINs should mostly be system generated. This example makes it seem like we're encouraging adding a FIN manually.

Reply:
That's a good point. I can try to find a better example for identifier type that works in sandbox because I still think the purpose of these examples should be to show what a maximal example body looks like.

Reply:
How would you feel about using Visit Id?

`            "type": {
                "coding": [
                    {
                        "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/319",
                        "code": "1081",
                        "display": "Visit Id"
                    }
                ],
                "text": "Visit Id"
            },`

Side note: I am also purposefully including the display even though on a write it isn't considered. Seems appropriate for a proprietary code.

Reply:
Seems like Visit Id could also be generated. I'm leaning back towards my original suggestion to not include it in the example.

If we are, I guess we can just stick with FIN.